flow/designs/asap7/ibex/rules-base.json

```diff
@@ -62,7 +62,7 @@
         },
         {
             "field": "finish__timing__setup__ws",
-            "value": 0.0,
+            "value": -41.67,
```

Comment:
This is ok but we should look at these new critical IO paths.

Reply:
Yes.  This will be an ongoing focus as we look into optimization. But we need a good baseline with good constraints.